openedx/adg/lms/webinars/helpers.py

```diff
@@ -30,9 +33,15 @@ def send_webinar_emails(template_slug, webinar, recipient_emails, send_at=None):
         'webinar_id': webinar.id,
         'webinar_title': webinar.title,
         'webinar_description': webinar.description,
-        'webinar_start_time': webinar.start_time.strftime("%B %d, %Y %I:%M %p %Z")
+        'webinar_start_time': webinar.start_time.strftime("%B %d, %Y %I:%M %p %Z"),
+        'webinar_link': webinar.meeting_link
```

Comment:
I hope you have updated the mandrill templates? Actually for reminder emails I have used `webinar_meeting_link`. Can we rename this too for this change? Also add trailing comma so that it does not show up in future if someone adds something to the context.

Reply:
Yes, have updated the webinar. `webinar_link` seems more consistent as we have used this name in other functions too.

Reply:
Then we might have to update the reminder templates and context to make it consistent all over the code base.